cli/golem-templates/templates/ts/ts-app-component-default/components-ts/component-name/src/main.ts

```diff
@@ -5,7 +5,7 @@ import {
     description,
 } from '@golemcloud/golem-ts-sdk';
 
-interface CustomData {
+type CustomData = {
```

Comment:
Is the SDK capturing properly the name of this too? (Will the AnalysedType have a name in the agent metadata)

Reply:
I have to set the name in AnalysedType. 

Yes SDK can see the alias name too, but someone can inline that object type without any name as well


`function get-weather(myData: {data: string, value: string})`

So should it be `myData` that should be the name of the type or the type alias name?